cypress/integration/inventory/actions-menu.spec.js

```diff
@@ -1,23 +1,36 @@
 import TopMenu from '../../support/fragments/topMenu';
 import Actions from '../../support/fragments/inventory/actions';
+import InventorySearchPanel from '../../support/fragments/inventory/inventorySearchPanel';
 
 describe('ui-inventory: actions', () => {
-  beforeEach('navigates to Actions', () => {
+  beforeEach('navigates to actions', () => {
     cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
     cy.visit(TopMenu.inventoryPath);
   });
 
   it('C196752 verifies action menu options before any search is conducted', () => {
-    const optionsShouldDisabled = [
-      Actions.saveUUIDOption,
-      Actions.saveCQLQueryOption,
-      Actions.exportMARCOption,
-      Actions.showSelectedRecordsOption
-    ];
+    cy.do(Actions.open());
 
-    Actions.open()
-      .then(() => {
-        optionsShouldDisabled.forEach(option => Actions.optionIsDisabled(option, true));
-      });
+    cy.expect([
+      Actions.saveUUIDsOption().is({ disabled: true }),
+      Actions.saveCQLQueryOption().is({ disabled: true }),
+      Actions.exportMARCOption().is({ disabled: true }),
+      Actions.showSelectedRecordsOption().is({ disabled: true }),
+    ]);
+  });
+
+  it('C196753 verifies action menu options after searching and selecting result', () => {
+    cy.do([
+      InventorySearchPanel.effectiveLocationInput.select([InventorySearchPanel.effectiveLocationValues.mainLibrary]),
+      InventorySearchPanel.firstResultCheckbox().click(),
```

Comment:
need to think, but can be useful to implement something like this: 
1. Read lines count
2. Randomly select 1 or 2 items(sometime only 1 row, in another runs - 2 rows) into the list of inventories
3. Check state of options into Actions menu

Reply:
For this test it is excess. We need to check only action options, not list of inventory.